tests/rptest/tests/partition_balancer_test.py

```diff
@@ -285,7 +286,15 @@ def test_movement_cancellations(self):
 
             self.run_validation(consumer_timeout_sec=CONSUMER_TIMEOUT)
 
-    @cluster(num_nodes=8, log_allow_list=LOG_ALLOW_LIST)
+    def check_rack_placement(self, topic, rack_layout):
```

Comment:
why is including this function in the backport necessary?

Reply:
not necessary. Don't understand how it appeared